### PR TITLE
ProgressBar: update demo description + minor code fix

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 1]
```diff
@@ -1 +1,16 @@
...
+
+- [min](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#max)    
+The min and max properties limit the range of accepted values. 
+
+- [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value)    
+This property specifies the current value. If you want to switch the ProgressBar to an indeterminate state, set the [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value) property to `false`.
+
+This demo uses the custom timer function to increase the ProgressBar value.
+
+When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event is raised. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/ProgressBar/Overview/description.md]: Avoidable passive voice.

```suggestion
When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event occurs. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.
```

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 2]
```diff
@@ -1 +1,16 @@
...
+
+The progress status displays a ratio between the current value and current value that indicates the made progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to change the status visibility. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/ProgressBar/Overview/description.md]: > The progress status displays a ratio between the current value and current value that indicates the made progress.

Something is amiss here. Inspect this sentence closely, for it makes no sense in its current state.

> Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to change the status visibility.

to change the status visibility == to display or hide the status string?

**Review by dmitry-eliseev-devexpress — COMMENTED** [review]
I must admit, you are making progress. However, something went wrong in the last paragraph.

File: JSDemos/Demos/ProgressBar/Overview/description.md (hunk 2 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/ProgressBar/Overview/description.md]: I am sorry, I will rewrite the paragrpah

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 3]
```diff
@@ -1 +1,16 @@
...
+The min and max properties limit the range of accepted values. 
+
+- [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value)    
+This property specifies the current value. If you want to switch the ProgressBar to an indeterminate state, set the [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value) property to `false`.
+
+This demo uses the custom timer function to increase the ProgressBar value.
+
+When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event occurs. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.
+
+The progress status displays a ratio between the current value and maximum value and indicates the made progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```
Comment (dmitry-eliseev-devexpress) [inline JSDemos/Demos/ProgressBar/Overview/description.md]: > a ratio between the current value and **maximum value**

An article is missing. Besides, we can join "current" and "maximum" under "value".

the made progress -> the progress made.

```suggestion
The progress status displays a ratio between the current and maximum values and indicates the progress made. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```

**Review by dmitry-eliseev-devexpress — DISMISSED** [review]
Now everything makes sense. Thank you.

File: JSDemos/Demos/ProgressBar/Overview/description.md (hunk 2 above)
Comment (dmitry-eliseev-devexpress) [reply JSDemos/Demos/ProgressBar/Overview/description.md]: Thank you.

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 4]
```diff
@@ -1 +1,16 @@
-The ProgressBar is&nbsp;a&nbsp;UI component that shows current progress.
\ No newline at end of file
+This demo shows how to configure the ProgressBar component.
+
+To create a ProgressBar, declare it in markup. You can specify the following properties to change the ProgressBar numeric scale:
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ProgressBar/Overview/description.md]:
```suggestion
To create a ProgressBar, declare it in markup. You can specify the following properties to change the ProgressBar's numeric scale:
```

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 5]
```diff
@@ -1 +1,16 @@
...
+
+- [min](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#min) and [max](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#max)    
+The min and max properties limit the range of accepted values. 
+
+- [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value)    
+This property specifies the current value. If you want to switch the ProgressBar to an indeterminate state, set the [value](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#value) property to `false`.
+
+This demo uses the custom timer function to increase the ProgressBar value.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ProgressBar/Overview/description.md]:
```suggestion
This demo uses the custom timer function to increase the ProgressBar's value.
```

File: JSDemos/Demos/ProgressBar/Overview/description.md [hunk 6]
```diff
@@ -1 +1,16 @@
...
+
+When the ProgressBar reaches the maximum value, the [complete](/Documentation/ApiReference/UI_Components/dxProgressBar/Events/#complete) event occurs. Use the [onComplete](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#onComplete) function to handle it.
+
+The progress status displays a ratio between the current and maximum values and indicates the progress made. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```
Comment (dirk-pieterse) [inline JSDemos/Demos/ProgressBar/Overview/description.md]:
```suggestion
The progress status displays a ratio between the current and maximum values and indicates the progress. Use the [showStatus](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#showStatus) property to display or hide the status string. To format the status string, use the [statusFormat](/Documentation/ApiReference/UI_Components/dxProgressBar/Configuration/#statusFormat) function.
```

**Review by dirk-pieterse — DISMISSED** [review]

**Pull request merged**